sigs.yml

```diff
@@ -350,8 +350,7 @@
     - name: Trask Stalnaker
       github: trask
   - name: Collector
-    meeting: Every Wednesday at 09:00 PT plus monthly on first Wednesday at 00:00 PT
-      and on third Wednesday at 16:00 PT
+    meeting: "Wednesdays, rotating between the following time slots: 00:00 UTC, 12:00 UTC, 16:00 UTC"
```

Comment:
I know this is counter the reason why this is done, but right now all meetings are scheduled as PT, we should look into that from a larger perspective, but would you mind to set up the meetings as "PT" for now? This is especially important for daylight saving time periods as they are regionally very different. UTC would mean that meeting times change 2 per year for those who have daylight saving times.

Reply:
I do not disagree that the way we write it down is not good for people outside the US, since I am affected as well, but it's scheduled that way right now. I am OK with this being the first UTC time, however when this gets scheduled the invite needs to be set as "UTC" as well such that now daylight saving times are respected, and we should also work towards fixing the other ones as well.

Reply:
I commit to setting the calendar entries to UTC, and will circle back after 6 months with the results of the "experiment". If we are happy with the results, we can start applying the same changes to other SIGs.

Reply:
thanks!